text/0001-poet2-consensus.md

```diff
@@ -0,0 +1,109 @@
+- Feature Name: PoET-2.0
+- Start Date: 2018-05-21
+- RFC PR: (leave this empty)
+- Sawtooth Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+As blockchain technology matures, there is a pressing need for more efficient consensus mechanisms that will scale to large networks without putting potentially crippling demands on the infrastructure required to support it. Proof of Elapsed Time (PoET) is an attempt at providing an innovative algorithm that not only scales very efficiently, but also places very few demands on the infrastructure.
+This document details a new mechanism for the PoET algorithm that overcomes some of the challenges with the original algorithm.
+The intended audience of this document is architects, developers and anyone who would like to get a better understanding of the details of the PoET 2.0 algorithm.
+
+This RFC describes a new PoET 2.0 Consensus for supporting the SGX PSE-free machines.
+
+# Motivation
+[motivation]: #motivation
+
+PoET 1.0 offers a very unique and efficient way to achieve consensus based on secure timers set inside an Intel® Software Guard Extensions (SGX) enclave. The secure timer and the monotonic counters used in PoET 1.0 rely on _SGX Platform Services_ to access these services from the system hardware. _SGX Platform Services_, unfortunately, are not yet available on all SGX enabled machines. While plans are in place for introducing these services universally across the processor lineup, it will take some time to release the services and for them to become ubiquitous. PoET 2.0 address these challenges by making the PoET algorithm independent of the _SGX Platform Services_.
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+In PoET 1.0, the PoET SGX enclave set a timer internally and would only release a WaitCertificate, once the timer elapsed. To be considered a candidate for consensus, a block (a.k.a Claim Block) would have to be accompanied by the corresponding WaitCertificate. This meant that peers receiving a Claim Block accompanied by the WaitCertificate, were assured that the block had not been forwarded prematurely, hence the phrase ‘Proof of Elapsed Time’.
```

Comment:
Nitpick: The term "claim block" is written inconsistently.  At first, it's "Claim Block". The rest of the paper uses "claim block", but there's also two occurrences of "ClaimBlock" without the space.

Technical writers prefer lower case (claim block), because it's easier to read. Plus, fewer terms would have to change. But "Claim Block" is acceptable as long as it's consistent.